Source/WebCore/page/ContextMenuController.cpp

```diff
@@ -1565,6 +1583,18 @@ void ContextMenuController::showImageControlsMenu(Event& event)
 
 #endif
 
+
+#if ENABLE(PDFJS)
+
+void ContextMenuController::performPDFJSAction(Frame& frame, const String& action)
+{
+    auto* pdfDocument = dynamicDowncast<PDFDocument>(frame.ownerElement()->document());
+    ASSERT(pdfDocument);
```

Comment:
This should either be RELEASE_ASSERT, or check pdfDocument for nullptr before using.